content/reference/current/user/binary/index.md

```diff
@@ -169,19 +169,51 @@ An artifact will be published will the `tests` classifier.
 
 ### SBT
 
+#### With the `test` configuration
+
+Add the following to your `build.sbt` configuration:
+
 ```scala
-packageBin in Test := (assembly in Test).value
-publishArtifact in Gatling := true
-publishTo :=
-	(if (isSnapshot.value)
-		Some("private repo" at "REPLACE_WITH_YOUR_SNAPSHOTS_REPOSITORY_URL")
-	else
-		Some("private repo" at "REPLACE_WITH_YOUR_RELEASES_REPOSITORY_URL")
+Test / packageBin := (Gatling / enterprisePackage).value
+Test / publishArtifact := true
```

Comment:
The `publish` task only publishes from the `Compile` and `Test` configuration scopes by default (based on the `publishArtifact` key), so setting `Gatling / publishArtifact` would do nothing. Otherwise the artifacts to publish must be manually configured, e.g. with `addArtifact`.

So we can either specify this:
```scala
Test / packageBin := (Gatling / enterprisePackage).value
Test / publishArtifact := true
```
or this:
```scala
Gatling / packageBin := (Gatling / enterprisePackage).value
addArtifact(Gatling / packageBin / artifact, Gatling / packageBin)
```

Reply:
(For tests in `it`, on the other hand, we don't have a choice - `publish` doesn't look for `IntegrationTest / publishArtifact` anyway. See sbt/sbt#2458.)

Reply:
Or alternately, I think I've seen how we could handle specifically `Gatling / publishArtifact := true` / `GatlingIt / publishArtifact := true` in our plugin, and then do the same thing as `addArtifact()` in that case.

Reply:
I think we should recommend a way that works both for Gatling and GatlingIt.

Your last suggestion looks nice. Have you tested it?

Reply:
I haven't yet tested the last one, but I _think_ I have figured out everything I need, so I'll test it now.

Reply:
So, what's the status?

Reply:
it works with the changes merged in the plugin with gatling/gatling-sbt-plugin#76 and this PR is updated to show that configuration.